node/pkg/watchers/sui/watcher.go

```diff
@@ -36,16 +31,31 @@ type (
 	// Watcher is responsible for looking over Sui blockchain and reporting new transactions to the wormhole contract
 	Watcher struct {
 		suiRPC           string
-		suiWS            string
 		suiMoveEventType string
 
 		unsafeDevMode bool
 
-		msgChan       chan<- *common.MessagePublication
-		obsvReqC      <-chan *gossipv1.ObservationRequest
-		readinessSync readiness.Component
+		msgChan                   chan<- *common.MessagePublication
+		obsvReqC                  <-chan *gossipv1.ObservationRequest
+		readinessSync             readiness.Component
+		latestProcessedCheckpoint int64
+		maximumBatchSize          int
+		descendingOrder           bool
+		loopDelayInSecs           int
```

Comment:
Make this a `time.Duration` then initialize it below with:
`loopDelayInSecs: time.Second`
Then you don't need to do that conversion every time you use it.

Reply:
Changed.